src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.UnitTests/Controllers/TrainingProgrammeControllerTests/WhenGettingStandardOptions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Api.Controllers;
+using SFA.DAS.CommitmentsV2.Api.Types.Responses;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetStandardOptions;
+using SFA.DAS.Testing.AutoFixture;
+using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.CommitmentsV2.Api.UnitTests.Controllers.TrainingProgrammeControllerTests
+{
+    public class WhenGettingStandardOptions
+    {
+        private Mock<IMediator> _mockMediator;
+
+        private TrainingProgrammeController _controller;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _mockMediator = new Mock<IMediator>();
+
+            _controller = new TrainingProgrammeController(_mockMediator.Object, Mock.Of<ILogger<TrainingProgrammeController>>());
+        }
+
+        [Test, MoqAutoData]
+        public async Task Then_GetStandardOptionsMediatorQueryIsCalled(string standardUId, GetStandardOptionsResult result)
+        {
+            _mockMediator.Setup(m => m.Send(It.IsAny<GetStandardOptionsQuery>(), CancellationToken.None)).ReturnsAsync(result);
```

Comment:
Can be combined with the below test if the other change is made - they cover the same thing

Reply:
Done